src/ui/base-page-signup.js

```diff
@@ -0,0 +1,140 @@
+/*
+For copyright information, see the AUTHORS.md file in the docs directory of this distribution and at
+https://github.com/fluid-project/sjrk-story-telling/blob/main/docs/AUTHORS.md
+
+Licensed under the New BSD license. You may not use this file except in compliance with this licence.
+You may obtain a copy of the BSD License at
+https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/main/LICENSE.txt
+*/
+
+"use strict";
+
+(function ($, fluid) {
+
+    // The signup page base grade
+    fluid.defaults("sjrk.storyTelling.base.page.signup", {
+        gradeNames: ["sjrk.storyTelling.base.page.authentication"],
+        pageSetup: {
+            authenticationUrl: "/signup"
+        },
+        validationErrorMapping: {
+            error_password_confirmation_blank: "properties.confirm.type",
+            error_password_confirmation_length: "properties.confirm.minLength",
+            error_password_confirmation_mismatch: "properties.confirmComparison.type"
+        },
+        model: {
+            confirmErrorMessage: "",
+            confirmComparisonErrorMessage: ""
+        },
+        modelListeners: {
+            confirmErrorMessage: {
+                this: "{authenticationUi}.dom.passwordConfirmationErrorText",
+                method: "text",
+                args: ["{change}.value"],
+                namespace: "confirmErrorMessageUpdate"
+            },
+            confirmComparisonErrorMessage: {
+                this: "{authenticationUi}.dom.passwordConfirmationComparisonErrorText",
+                method: "text",
+                args: ["{change}.value"],
+                priority: "after:confirmErrorMessageUpdate",
+                namespace: "confirmComparisonErrorMessageUpdate"
+            }
+        },
+        listeners: {
+            "onAuthenticationRequested.resetPasswordConfirmationErrorMessage": {
+                func: "{that}.resetPasswordConfirmationErrorMessage",
```

Comment:
The comment above about bypassing the extra layer of invokers applies to here and line 50 too.